bin/companion

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+sh ../env.sh
+
+export UPPYSERVER_DATADIR="./packages/@uppy/companion/output"
```

Comment:
We may want to put this in the repo root instead so we can never accidentally publish it.

Reply:
fixed